### ISSUE TITLE
Wiring / Schematic

### ISSUE THREAD

**Issue by schoofseggl** [scott-vincent/autopilot-panel#1]
Hi, 

Do you have any schematics for the wiring of the raspi zero w gpios for the autopilot panel knobs and switches? I can't find any information on this in this repo... 

Regarding interfacing to MSFS: Does the AP panel use the SDK or the VJOY input simulator? 

Cheers Andreas

**Comment by scott-vincent** [comment]
No, because you can wire it however you like. All the pin numbers are in the settings/autopilot-panel.json file. See the sevensegment.cpp for a large comment explaining how to wire the 7 segment displays up. Then all you need is any 3 inputs for each rotary encoder and an input + output for each push button led.

**Comment by scott-vincent** [comment]
AP uses the SDK indirectly (via instrument-data-link). It doesn't need vJoy.
